src/Console.php

```diff
@@ -163,6 +177,8 @@ public function send($js)
         return $this;
     }
 
+    public $last_exit_code = null;
```

Comment:
@var and description ?

Reply:
changed to lastExitCode, moved up and added comment.